src/guides/reset_with_sd_card.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: troubleshooting-guide
+description: Home Assistant Green
+guideName: Resetting the device using an SD card
+tools:
+  title: Prerequisites
+  image: green_sd_card.png
+  content: |
+    <ul>
+        <li>Home&nbsp;Assistant Green</li>
+        <li>Home Assistant 13 or older and you cannot upgrade to Home Assistant 13.1 or later</li>
```

Comment:
This procedure works with any HAOS version, and there is no limitation on what version we can upgrade to. So drop this sentence.

```suggestion
```